src/Plugin/RulesAction/RulesComponentAction.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Plugin\RulesAction\RulesComponentAction.
+ */
+
+namespace Drupal\rules\Plugin\RulesAction;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\rules\Core\RulesActionBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a generic 'Execute Rules component' action.
+ *
+ * @RulesAction(
+ *   id = "rules_component",
+ *   deriver = "Drupal\rules\Plugin\RulesAction\RulesComponentActionDeriver"
+ * )
+ *
+ * @todo: Add access callback information from Drupal 7.
```

Comment:
hm, it does not look so - since we moved away from core actions we lost support for access() also. Looks like we need to work on that.